src/ophys_etl/workflows/ophys_experiment.py

```diff
@@ -85,3 +91,114 @@ def from_id(
+        """
+
+        workflow_step_run_id = get_latest_run(session,
+                                              WorkflowStepEnum.MOTION_CORRECTION,
+                                              WorkflowNameEnum.OPHYS_PROCESSING,
+                                              )
+        query = (
+            select(
+                MotionCorrectionRun,
+            )
+            .where(MotionCorrectionRun.workflow_step_run_id == workflow_step_run_id) 
+        )
+
+        result = session.execute(query).all()
+        motion_border = result[0][0]
+        return {
+            "x0": motion_border.max_correction_left,
+            "x1": motion_border.max_correction_right,
+            "y0": motion_border.max_correction_up,
+            "y1": motion_border.max_correction_down
+        }
+
+    def _generate_binary_mask(self, ophys_roi: OphysROI, ophys_roi_mask_values: List[OphysROIMaskValue]) -> List[List[bool]]:
+        """
+        Generate binary mask for an ROI
+
+        Parameters
+        ----------
+        ophys_roi
+            The ophys ROI
+        ophys_roi_mask_values
+            The ophys ROI mask values
+
+        Returns
+        -------
+        List[List[bool]]
+            A list of lists of booleans representing the binary mask
+        """
+        mask = np.zeros((ophys_roi.height, ophys_roi.width), dtype=bool)
+        for ophys_roi_mask_value in ophys_roi_mask_values:
+            mask[ophys_roi_mask_value.row_index, ophys_roi_mask_value.col_index] = True
+        return mask.tolist()
+    
+    def get_ophys_experiment_roi_metadata(
+            self,
+            session: Session) -> List[Dict]:
+        """
+        Get ROI metadata for an ophys experiment
+        
+        Parameters
+        ----------
+        ophys_experiment_id
+            The ophys experiment id
+        session
+            The database session
+
+        Returns
+        -------
+        List[Dict]
+            A list of dictionaries containing ROI metadata
+        """
+        workflow_step_run_id = get_latest_run(session,
+                                              WorkflowStepEnum.MOTION_CORRECTION,
+                                              WorkflowNameEnum.OPHYS_PROCESSING,
+                                              self.id,
+                                              )
+        query = (
+            select(
+                OphysROI
+            )
+            .where(OphysROI.workflow_step_run_id == workflow_step_run_id)
+        )
+
+        result = session.execute(query).all()
+        roi_metadata = []
+        for ophys_roi in result:
+            ophys_roi = ophys_roi[0]
+            query = (
+            select(
+                OphysROIMaskValue
+            )
+            .where(OphysROIMaskValue.ophys_roi_id == ophys_roi.id)
+            )
+            masks_list = session.execute(query).all()
+            masks_list = [mask[0] for mask in masks_list]
```

Comment:
can you clarify what `mask[0]` is selecting here?

Reply:
Each `mask` that's iterated in `masks_list` is a tuple of size 1. I need to select the object from the tuple.
`(OphysROIMaskValue(col_index=6, ophys_roi_id=1, id=1, row_index=5),)`